package.json

```
@@ -27,6 +27,7 @@
     "index.js"
   ],
   "devDependencies": {
+    "@types/tape": "^4.0.0",
```

missing tape types were hiding exception with return type

---

oh that’s interesting